  Header header
  string loc_type
  string mavName
  geometry_msgs/Point position